app/src/main/java/ch/sdp/vibester/activity/CreateProfileActivity.kt

```diff
@@ -41,4 +63,23 @@ class CreateProfileActivity : AppCompatActivity() {
 
         startActivity(newIntent)
     }
+
+    private fun uploadImage() {
+        val intent = Intent(Intent.ACTION_PICK)
+        intent.type = "image/*"
+        startActivityForResult(intent, REQUEST_CODE)
+    }
+
+
+    private fun updateUI() {
+        findViewById<TextView>(R.id.uploadStatus).text = "Upload OK"
```

Comment:
You can use strings.xml resources here instead of hardcoding. Let's keep it clean.

Reply:
Agreed.